### PR TITLE
podman-remote save [image]

### PR DESCRIPTION
**Pull request by baude**
Add the ability to save an image from the remote-host to the
remote-client.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/save.go [hunk 1]
```diff
@@ -52,7 +41,7 @@ func init() {
 	saveCommand.SetUsageTemplate(UsageTemplate())
 	flags := saveCommand.Flags()
 	flags.BoolVar(&saveCommand.Compress, "compress", false, "Compress tarball image layers when saving to a directory using the 'dir' transport. (default is same compression type as source)")
-	flags.StringVar(&saveCommand.Format, "format", "", "Save image to oci-archive, oci-dir (directory with oci manifest type), docker-dir (directory with v2s2 manifest type)")
+	flags.StringVar(&saveCommand.Format, "format", "docker-archive", "Save image to oci-archive, oci-dir (directory with oci manifest type), docker-dir (directory with v2s2 manifest type)")
```
Comment (mheon) [inline cmd/podman/save.go]: The default here isn't one of the listed options?

File: pkg/varlinkapi/images.go [hunk 2]
```diff
@@ -739,3 +739,107 @@ func (i *LibpodAPI) ImagesPrune(call iopodman.VarlinkCall, all bool) error {
 	}
 	return call.ReplyImagesPrune(prunedImages)
 }
+
+// ImageSave ....
+func (i *LibpodAPI) ImageSave(call iopodman.VarlinkCall, options iopodman.ImageSaveOptions) error {
+	// TODO start here
```
Comment (mheon) [inline pkg/varlinkapi/images.go]: ?

File: cmd/podman/save.go (hunk 1 above)
Comment (baude) [reply cmd/podman/save.go]: Nope, thats the way the help was dude.  But, if you look at the help, cobra does do this which to me is the best of all worlds:

```--format string   Save image to oci-archive, oci-dir (directory with oci manifest type), docker-dir (directory with v2s2 manifest type) (default "docker-archive")
```

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**